pyhive/hive.py

```diff
@@ -66,11 +66,14 @@ def connect(*args, **kwargs):
 class Connection(object):
     """Wraps a Thrift session"""
 
-    def __init__(self, host, port=10000, username=None, database='default', auth='NONE',
+    def __init__(self, host, kerberos_service_name=None, port=10000, username=None, password=None, database='default', auth='NONE',
```

Comment:
Inserting arguments is backwards incompatible for anyone not using named arguments)